src/SFML/Audio/SoundFileWriterWav.cpp

```diff
@@ -137,8 +137,7 @@ bool SoundFileWriterWav::writeHeader(unsigned int sampleRate, unsigned int chann
     m_file.write(mainChunkId, sizeof(mainChunkId));
 
     // Write the main chunk header
-    Uint32 mainChunkSize = 0; // placeholder, will be written later
-    encode(m_file, mainChunkSize);
+    encode(m_file, static_cast<Uint32>(0)); // 0 is a placeholder, will be written later
```

Comment:
Does `0` really need casting? What about using `0u`?

Reply:
I added that because there are many overloads of `encode` that expect different types for that 2nd parameter and I wanted to be absolutely sure the same overload was being called. `0u` would be correct on many platforms but I don't believe it's guaranteed that `0u` will be 32 bits wide so I went with the static cast to be safe.

Reply:
Ah right, didn't remember that here are different overloads, in that case it makes sense.
Thanks for clarifying. 🙂